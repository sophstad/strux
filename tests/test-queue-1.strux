void main() {
	Queue<string> q;
	q = new Queue<string>();
}