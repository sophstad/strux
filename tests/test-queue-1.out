hello
1.000000
1
1
